lib/denv.rb

```diff
@@ -34,7 +36,11 @@ class << self
     def load(filename = DEFAULT_ENV_FILENAME)
       filename = File.expand_path(filename.to_s)
       run_callback(filename) do
-        ENV.update(build_env(filename))
+        env = build_env(filename)
+        previous_keys = (ENV[SPECIAL_KEY] || '').split(DELIMITER)
+        ENV.delete_if {|k, _| previous_keys.include?(k) } unless previous_keys.empty?
```

Comment:
@k0kubun I guess it won't because `previous_keys ` will be empty array in the first execution.

```ruby
previous_keys = ''.split('=') #=> []
previous_keys.empty? #=> true
h = { 'OK' => 1 } #=> {"OK"=>1}
h.delete_if {|k, _| previous_keys.include?(k) } #=> {"OK"=>1}
h #=> {"OK"=>1}
```

How do you think? Something wrong?